packages/clerk-js/package.json

```diff
@@ -50,9 +50,13 @@
   },
   "browserslist": "last 2 versions, ios_saf > 12, Safari > 12, > 1%, not dead, not ie > 0",
   "dependencies": {
+    "@clerk/localizations": "3.1.1",
+    "@clerk/shared": "2.8.5",
+    "@clerk/types": "4.24.0",
```

Comment:
[nit] Non-module packages, like types, are dev deps so they are not needed to be installed by consumers. Should be safe to move this to `devDeps`

Reply:
I was just looking at `@clerk/types` and it isn't actually a types-only package as it exports a JS module. That's a bit confusing. @BRKalow thoughts?